astronomer/providers/google/cloud/triggers/gcs.py

```diff
@@ -26,16 +29,25 @@ def __init__(
         object_name: str,
         poke_interval: float,
         google_cloud_conn_id: str,
-        hook_params: Dict[str, Any],
+        hook_params: dict[str, Any],
     ):
+        warnings.warn(
+            (
+                "This class is deprecated and will be removed in 2.0.0."
```

Comment:
same question to check if we would need to add this to the docstring of the class too?

Reply:
yep, sounds good! just added